packages/flutter/lib/src/material/bottom_navigation_bar.dart

```diff
@@ -34,6 +34,31 @@ enum BottomNavigationBarType {
   shifting,
 }
 
+/// Refines the layout of a [BottomNavigationBar] when the enclosing
+/// [MediaQueryData.orientation] is [Orientation.landscape].
+enum BottomNavigationBarLandscapeLayout {
```

Comment:
That is quite the mouthful 😄.

Reply:
I tried to come up with something shorter however it seemed necessary to use the class name as a prefix.

Reply:
I totally get that, and don't have a better suggestion. Just struck me as really long, but sometimes that is needed.